test/gscraper_web/views/layout_view_test.exs

```diff
@@ -1,5 +1,5 @@
-defmodule GscraperWebWeb.LayoutViewTest do
-  use GscraperWebWeb.ConnCase, async: true
+defmodule GscraperWeb.LayoutViewTest do
```

Comment:
Should we remove this test? It does not contain any test case.

Reply:
Test added in bb51c11.